breizorro/breizorro.py

```diff
@@ -145,6 +147,8 @@ def main():
                         help='Merge in one or more masks or region files')
     parser.add_argument('--subtract', dest='subtract', metavar="MASK(s)|REG(s)", nargs='+',
                         help='Subract one or more masks or region files')
+    parser.add_argument('--merge_and', dest='mergeand', action='store_true',
```

Comment:
In general, I would prefer ``--merge-and`` over ``--merge_and`` -- but thinking about it another minute, it's not really a "merge" operation, it's an "intersect"... I think the CLI would be cleaner if this was defined as a separate top-level operation rather than a flag on top of "merge":

```python
  parser.add_argument('--merge', dest='merge', metavar="MASK(s)|REG(s)", nargs='+',
                        help='Merge in one or more masks or region files')
  parser.add_argument('--subtract', dest='subtract', metavar="MASK(s)|REG(s)", nargs='+',
                        help='Subract one or more masks or region files')
  parser.add_argument('--intersect', dest='intersect', metavar="MASK(s)|REG(s)", nargs='+',
                        help='Intersect with one or more masks or region files')
```

Reply:
Why don't we just implement explicit, top level logical OR / AND / XOR operations? I think the meaning of "intersect" is less clear. Wrote a simple script for this ages ago and have used it more times than I can count, so it's probably worth adding this general functionality to breizorro...

https://github.com/IanHeywood/oxkat/blob/master/tools/image_logic.py

Reply:
Set theory language vs. boolean logic language, take your pick... 

But then ``add_argument`` does allow multiple option names, so why not have it both ways:

```python
parser.add_argument('--merge', '--or', dest='merge', metavar="MASK(s)|REG(s)", nargs='+',
                        help='Merge (logical-OR) one or more masks or region files')
```

Just not sure what ``--subtract`` is in boolean language? It's not XOR (and is XOR conceivably useful for masks anyway?)

Reply:
I think subtract is subtract...

Reply:
> Just not sure what --subtract is in boolean language?

NAND? 🤔

> is XOR conceivably useful for masks anyway?

Probably not (that's not not NOT) actually.